### PR TITLE
Do not multiply mean coverage values by 100 on coverage reports

### LINKED ISSUE #231: Mean sample in coverage in coverage report is way too high
I think it gets multiplied by 100 together with the numbers of the same row, because the others are percentages. Will fix!

image

### PR DESCRIPTION
**Pull request by northwestwitch**
### This PR adds | fixes:
- Fix #231 

### How to test:
- Evaluate mean gene coverage over demo d4 file and demo bed file with 4 genes genes using d4toold (direct call) and make sure coverage is around 20
- Test mean coverage using this branch

### Expected outcome:
- Mean coverage in report should be 100 times lower

### Review:
- [x] Code approved by HS
- [x] Tests executed by CR

This [version](https://semver.org/) is a:
- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [x] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

**Comment by northwestwitch** [comment]
Real coverage over each gene of the demo panel is 22:

image

Mean coverage for sample in report is 18 with the fix. The difference is probably due to the fact that the report takes into account only the transcripts of the gene and not the entire gene intervals as in the case above with d4tools:

image

File: src/chanjo2/meta/handle_report_contents.py [hunk 1]
```diff
@@ -205,24 +205,20 @@ def get_report_completeness_rows(
...
 
         completeness_row: Dict[str, float] = {}
         for completeness_key, completeness_values in sample_stats.items():
-            completeness_row[completeness_key] = (
-                round((mean(completeness_values) * 100), 2)
-                if completeness_values
-                else 0
-            )
+            column_value = mean(completeness_values) if completeness_values else 0
+            if completeness_key != "mean_coverage":
```
Comment (northwestwitch) [inline src/chanjo2/meta/handle_report_contents.py:219]: This is the real fix

File: src/chanjo2/meta/handle_report_contents.py [hunk 2]
```diff
@@ -205,24 +205,20 @@ def get_report_completeness_rows(
     completeness_rows: List[str, Dict[str, float]] = []
 
     for sample, interval_stats in samples_coverage_stats.items():
-        sample_stats: Dict[str, List[float]] = {"mean_coverage": []}
+        sample_stats: Dict[str, List[float]] = {
+            "mean_coverage": [interval.mean_coverage for interval in interval_stats]
+        }
         for level in levels:
-            sample_stats[f"completeness_{level}"]: List[float] = []
```
Comment (northwestwitch) [inline src/chanjo2/meta/handle_report_contents.py:210]: Here is just a refactor for simplifying the code

**Review by henrikstranneheim — APPROVED** [review]
👍

**Pull request merged**